src/main/java/uk/gov/digital/ho/hocs/casework/domain/model/CaseDataTagId.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.digital.ho.hocs.casework.domain.model;
+
+import java.io.Serializable;
+import java.util.Objects;
+import java.util.UUID;
+
+public class CaseDataTagId implements Serializable {
+
+    private UUID caseUuid;
+
+    private String tag;
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {return true;}
+        if (o == null || getClass() != o.getClass()) {return false;}
+        CaseDataTagId that = (CaseDataTagId) o;
+        return Objects.equals(caseUuid, that.caseUuid) && Objects.equals(tag, that.tag);
+    }
```

Comment:
This is essentially duplicated as in line 66 onwards in `src/main/java/uk/gov/digital/ho/hocs/casework/domain/model/CaseDataTag.java` . Maybe some potential for reuse as it's a pretty generic method?

Reply:
This is boilerplate equals implementation that IntelliJ gives you out the box. I'd prefer to keep the equals and hashcode stuff close to the class incase it ever needs to differ